ghcide/src/Development/IDE/GHC/CoreFile.hs

```diff
@@ -228,3 +230,45 @@ tc_iface_bindings (TopIfaceNonRec v e) = do
 tc_iface_bindings (TopIfaceRec vs) = do
   vs' <- traverse (\(v, e) -> (,) <$> pure v <*> tcIfaceExpr e) vs
   pure $ Rec vs'
+
+-- | Prefixes that can occur in a GHC OccName
+occNamePrefixes :: [T.Text]
+occNamePrefixes =
```

Comment:
Is this needed now?

Reply:
This is a refactor to allow the `prefixes` from the `Logic` module available to other plugins. Thanks to your suggestion, we now don't need to access to those prefixes from the `AtPoint` module now but other modules might need access to the prefixes in the future. We can revert the refactor, or if it's considered an improvement we can keep it. I'll defer to you and others with more experience on the codebase.